Source/WebCore/fileapi/URLKeepingBlobAlive.h

```diff
@@ -62,3 +63,19 @@ class URLKeepingBlobAlive {
 };
 
 } // namespace WebCore
+
+namespace WTF {
+template<> struct CrossThreadCopierBase<false, false, WebCore::URLKeepingBlobAlive> {
```

Comment:
I don't fully understand why we need this template specialization. Likely related to the fact that the class is not copyable but I still don't get why. It's a bit unfortunate to have to do this.

Reply:
Right, this is not needed any more (needed for previous iteration)